Vagrantfile

```diff
@@ -1,27 +1,48 @@
-debug = true
+# options for Vagrant provisioning; do NOT commit personal changes to these
+provisioning = {
+
+  # if enabled, the base Ubuntu box will be used and Ansible will provision it;
+  # when disabled, the custom Korobi box will be used and no provisioning will
+  # happen; this is useful it you find yourself using `vagrant destroy -f;
+  # vagrant up` very often, or if you're on Windows, which doesn't support
+  # Ansible
+  enabled: true,
+
+  # if true, Ansible's verbose mode (level 'vv') will be used; does nothing if
+  # provisioning is disabled
+  debug: false
+  
+}
 
 Vagrant.configure(2) do |config|
+  if provisioning[:enabled]
     config.vm.box = 'ubuntu/trusty64'
-    config.vm.hostname = 'korobi'
+  else
+    config.vm.box = 'bionicrm/korobi-web'
```

Comment:
windows users don't, but Linux users can still do that, right?

I'm just not seeing the use of the different boxes when one of them is a timesaver because most of the work is already done once in comparison to the other.